data/roles.js

```diff
@@ -852,81 +878,85 @@ const roleData = {
         "Amnesiac": {
             alignment: "Independent",
             description: [
-                "Once per game can become the role of a dead player."
+                "Choose to become the role of a dead player, once per game."
             ],
         },
         "Survivor": {
             alignment: "Independent",
             description: [
-                "Wins if survives until the end of the game."
+                "Wins if alive at the end of the game."
             ],
         },
         "Alien": {
             alignment: "Independent",
             description: [
-                "Can choose one player to probe each night.",
+                "Chooses one player to probe each night.",
                 "Wins if all players left alive have been probed."
             ],
         },
         "Old Maid": {
             alignment: "Independent",
             description: [
-                "Can choose one player to swap roles with each night.",
+                "Chooses a player to swap roles with each night.",
+                "Chosen player becomes the Old Maid.",
                 "Cannot win the game.",
             ],
         },
         "Traitor": {
             alignment: "Independent",
             description: [
-                "Wins when mafia wins.",
+                "Wins with mafia.",
                 "Does not count towards mafia win count.",
             ],
         },
         "Mastermind": {
             alignment: "Independent",
             description: [
-                "Mafia meeting is anonymous",
-                "Wins instead of mafia and counts toward their total",
+                "Mafia meeting is anonymous if Mastermind is present in the game.",
+                "Wins instead of mafia and counts toward their total.",
                 ],
         },
         "Lover": {
             alignment: "Independent",
             description: [
-                "Once per game can fall in love with another player.",
-                "Both players die if either dies.",
+                "Fall in love with another player, once per game.",
+                "Both players die if either of them is killed.",
```

Comment:
Grammar, I think this should be "either of them are killed" instead of "either of them is killed."